README.md

```diff
@@ -1,24 +1,109 @@
-# README
+# Hello-rails-react
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+The is an all in one react-rails app showing how the two can integrate. 
 
-Things you may want to cover:
 
-* Ruby version
+## Built With
 
-* System dependencies
+- Webpack
+- Ruby on Rails
+- React
+- Redux
 
-* Configuration
 
-* Database creation
 
-* Database initialization
+## Getting Started
 
-* How to run the test suite
+To get a local copy up and running follow these simple example steps.
 
-* Services (job queues, cache servers, search engines, etc.)
+- git clone https://github.com/assadounto/hello-rails-react.git
 
-* Deployment instructions
+- cd hello-rails-react
 
-* ...
+
+### Prerequisites
+
+Ruby
+Rails
+PostgresSQL
+
+
+### Setup
+
+Install gems with:
+
+```
+bundle install
+```
+
+Setup database with:
+
+```
+rails db:create
+rails db:migrate
+```
+
+Start server with:
+
+```
+foreman start -f Profile.dev
```

Comment:
- [ ] It would be better if following the README file we can be able to run and start the app locally. However, I'm unable to run the app on my machine as you can see in the screenshot below
![Screenshot from 2022-10-31 18-39-15](https://user-images.githubusercontent.com/46388113/199085381-3ddb8ec7-6322-4621-be74-9d70d4c32fba.png)

In the last command as we have in the README, there is no proc file to run the command

Reply:
Sorry it is a 
typo @addod19 please try ```foreman start -f Procfile.dev```